Automattic/jetpack
# Convert 'types' attr string into array for multiple post type support

Fixes #25740

#### Changes proposed in this Pull Request:
* Change will allow support of multiple post types for "Top posts and Pages Widget shortcode".

#### Other information:


#### Does this pull request change what data or activity we track or use?
No


#### Testing instructions:

* Go to post editor, add shortcode `[jetpack_top_posts_widget types="post,page"]` and save the post.
* Open post frontend. It should display posts including post types `post` and `page`.
